.env.production

```diff
@@ -1,2 +1 @@
-REACT_APP_TEST=test
```

Comment:
Delete these two variables because they are not used throughout the project, and they are not intended to be used.